src/test/regress/expected/views_create.out

```diff
@@ -0,0 +1,63 @@
+CREATE SCHEMA views_create;
+SET search_path TO views_create;
+CREATE TABLE view_test_table(a INT NOT NULL PRIMARY KEY, b BIGINT, c text);
+CREATE OR REPLACE VIEW select_filtered_view AS
+    SELECT * FROM view_test_table WHERE c = 'testing'
+    WITH CASCADED CHECK OPTION;
+CREATE OR REPLACE VIEW select_all_view AS
+    SELECT * FROM view_test_table
+    WITH LOCAL CHECK OPTION;
+CREATE OR REPLACE VIEW count_view AS
+    SELECT COUNT(*) FROM view_test_table;
+SELECT create_distributed_table('view_test_table', 'a');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+INSERT INTO view_test_table VALUES (1,1,'testing'), (2,1,'views');
+SELECT * FROM count_view;
+ count
+---------------------------------------------------------------------
+     2
+(1 row)
+
+SELECT COUNT(*) FROM count_view;
+ count
+---------------------------------------------------------------------
+     1
+(1 row)
+
+SELECT COUNT(*) FROM select_all_view;
+ count
+---------------------------------------------------------------------
+     2
+(1 row)
+
+SELECT * FROM select_filtered_view;
+ a | b |    c
+---------------------------------------------------------------------
+ 1 | 1 | testing
+(1 row)
+
+-- dummy temp recursive view
+CREATE TEMP RECURSIVE VIEW recursive_defined_non_recursive_view(c) AS (SELECT 1);
+CREATE MATERIALIZED VIEW select_all_matview AS
+    SELECT * FROM view_test_table
+    WITH DATA;
+CREATE MATERIALIZED VIEW IF NOT EXISTS select_filtered_matview AS
+    SELECT * FROM view_test_table WHERE c = 'testing'
+    WITH NO DATA;
+REFRESH MATERIALIZED VIEW select_filtered_matview;
+SELECT COUNT(*) FROM select_all_matview;
+ count
+---------------------------------------------------------------------
+     2
+(1 row)
+
+SELECT * FROM select_filtered_matview;
+ a | b |    c
```

Comment:
should we join 2 views that select from distributed tables (and maybe join with a distributed table as well?)